.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -194,33 +194,6 @@ jobs:
         env:
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
-      - name: Post success to Slack channel dev-connectors-extensibility
```

Comment:
Do we not still want to post success to Slack on `publish-cdk`?

Reply:
That's a good question. For me now with the added publishing of the Docker image, I felt that a release is not a success until both of these steps were done. 

Example: Given a workflow that is able to push to pypi and not Dockerhub, I probably want just one message and it is that it failed. In the message, there is a link to the GitHub run and I can easily see which step failed. One thing that might be troublesome in that case is that the pipeline currently does not allow to re-run the pipeline without publishing to pypi so I'll have to bump the version again which is a little bit annoying...

Reply:
👍 makes sense to me.